2.crud_queries/tasks.md

```diff
@@ -1,31 +1,70 @@
 1. Create a database named `blog`.
+use blog
 
 2. Create a collection called 'articles'.
+db.createCollection("articles")
 
 3. Insert multiple documents(at least 3) into articles. It should have fields
+db.articles.insert({titel: "Yugandhar", author:"Shivaji Sawant", age:65, details:"It is based on life of Krish
+na"})
+
+db.articles.insert({titel: "Chawa", author:"Shivaji Sawant", age:65, details:"It is based on life of  Sambhaji
+, one of the grea Maratha  warrior"})
+
+db.articles.insert({titel: "Mrityunjay", author:"Shivaji Sawant", age:65, details:"It is based on life of Karn
+a"})
+
+db.articles.insert({titel: "Panipat", author:"Vishwas Patil", age:60, details:"Third battsl of Panipat"})
+
+
 
 4. Find all the articles using `db.COLLECTION_NAME.find()`
 
+db.articles.find()
+{ "_id" : ObjectId("5ea6fd15af7067671580c2d5"), "titel" : "Chawa", "author" : "Shivaji Sawant", "age" : 65, "details" : "It is based on life of  Sambhaji, one of the grea Maratha  warrior" }
+{ "_id" : ObjectId("5ea6ffe4af7067671580c2d6"), "titel" : "Yugandhar", "author" : "Shivaji Sawant", "age" : 65, "details" : "It is based on life of Krishna" }
+{ "_id" : ObjectId("5ea70033af7067671580c2d7"), "titel" : "Mrityunjay", "author" : "Shivaji Sawant", "age" : 65, "details" : "It is based on life of Karna" }
+{ "_id" : ObjectId("5ea704f7af7067671580c2d8"), "titel" : "Panipat", "author" : "Vishwas Patil", "age" : 60, "details" : "Third battsl of Panipat" }
+
+
 5. Find a document using _id field.
 
+db.articles.find({_id:ObjectId("5ea704f7af7067671580c2d8")})
+
+
+
 6. Find documents using title and author's name field.
```

Comment:
You have to query document using title and author.
I didn't say to return title and author.
It should be:-
```
db.articles.find({title: '', author: ''});
```